tests/dialects/test_doris.py

```diff
@@ -18,3 +18,56 @@ def test_regex(self):
                 "doris": "SELECT REGEXP(abc, '%foo%')",
             },
         )
+
+    def test_concat(self):
+        self.validate_all(
+            "SELECT a || b",
+            write={
+                "doris": "SELECT CONCAT(a, b)",
+            },
+        )
+
+    def test_to_date(self):
+        self.validate_all(
+            "SELECT TO_DATE('2019-12-01')",
+            read={
+                "oracle": "SELECT TO_DATE('2019-12-01')",
+                "hive": "SELECT TO_DATE('2019-12-01')",
+            },
+            write={
+                "doris": "SELECT TO_DATE('2019-12-01')",
+            },
+        ),
+        self.validate_all(
+            "SELECT STR_TO_DATE('2019-12-01', 'yyyy-MM-dd')",
+            read={
+                "oracle": "SELECT TO_DATE('2019-12-01', 'yyyy-MM-dd')",
+            },
+            write={
+                "doris": "SELECT STR_TO_DATE('2019-12-01', 'yyyy-MM-dd')",
+            },
+        ),
+        self.validate_all(
+            "SELECT STR_TO_DATE('2019-12-01', '%Y-%m-%d')",
+            write={
+                "doris": "SELECT STR_TO_DATE('2019-12-01', '%Y-%m-%d')",
+            },
+        ),
+        self.validate_all(
+            "NVL2(x, y, z)",
+            write={
+                "doris": "CASE WHEN x IS NOT NULL THEN z ELSE y END",
+            },
+        ),
+        self.validate_all(
+            "to_char('2009-10-04 22:23:00', 'yyyymm')",
+            write={
+                "doris": "DATE_FORMAT('2009-10-04 22:23:00', 'yyyymm')",
+            },
+        ),
+        self.validate_all(
+            "to_char(t, 'FM99999999990.00')",
+            write={
+                "doris": "Round(t,2)",
+            },
+        ),
```

Comment:
It is precisely because there is no to_char function in doris, so it is implemented in other ways. Here I do not implement it in a standardized way, and adopt a relatively stupid way. Convert the to_char function into two different forms, one is the formatting form of the date, and the other is the formatting form of the value. I hope you can provide me with a better solution

Reply:
Sorry, I don't follow. From what I can understand, you want to convert `TO_CHAR` from some dialect into Doris, which doesn't support that function. What is the input dialect in this case?

Reply:
The input dialect is oracle，How should I get in touch with you? Is there any other way of communication?